content/tutorials/reactor-event-google.textile

```diff
@@ -34,19 +34,19 @@ meta_keywords: "Ably realtime, pizza ordering app, Google Cloud Functions, integ
+With the rise of serverless solutions such as "Google Cloud Functions":https://cloud.google.com/functions/, the problem goes away. Create server functions that contain your business logic, credentials and other sensitive data, hook them up to your application, and all of the sensitive work is hidden away from the user, not to mention being managed by world-class infrastructure providers such as Amazon, Google or Microsoft.
 
-Ably's *"Reactor Events":https://ably.com/integrations* feature provides a way to define custom criteria that will automatically trigger different kinds of actions in response to channel messages, "channel lifecycle events":/realtime/channel-metadata#lifecycle-events (such as a channel being created) and presence events (such as users entering or leaving a channel), and is how we'll integrate with your server functions without exposing them to your users.
+Ably's *"Webhooks":https://ably.com/integrations* feature provides a way to define custom criteria that will automatically trigger different kinds of actions in response to channel messages, "channel lifecycle events":/realtime/channel-metadata#lifecycle-events (such as a channel being created) and presence events (such as users entering or leaving a channel), and is how we'll integrate with your server functions without exposing them to your users.
 
 h2. What we'll be building
 
-For this tutorial, we'll be using *"Reactor Function":/general/events* events in response to incoming messages in order to *automatically invoke your own Google functions*. We're going to build a simple pizza ordering system with a virtual pizza assistant chat bot. When a new customer hits your website, they'll be assigned a random ID, and their messages will be published to a private channel created just for that customer. A Reactor Event triggered by those messages will call your Google functions to invoke the pizza chat bot to take their order. Using the Ably REST API, it will post its responses back on another channel that the web application subscribes to.
+For this tutorial, we'll be using *"Webhooks":/general/events* events in response to incoming messages in order to *automatically invoke your own Google functions*. We're going to build a simple pizza ordering system with a virtual pizza assistant chat bot. When a new customer hits your website, they'll be assigned a random ID, and their messages will be published to a private channel created just for that customer. A webhook triggered by those messages will call your Google functions to invoke the pizza chat bot to take their order. Using the Ably REST API, it will post its responses back on another channel that the web application subscribes to.
```

Comment:
> `*"Webhooks":/general/events*`

I'm not sure, but was there a reason this link was bolded? If not let's just make it a link.